src/components/appointments/CreateAppointment/Summary.js

```diff
@@ -38,7 +38,8 @@ const createAppointment = async (baseUrl, submission, appointmentData, privacyPo
     contactDetails,
     privacyPolicyAccepted,
   };
-  return await post(`${baseUrl}appointments/appointments`, body);
+  const response = await post(`${baseUrl}appointments/appointments`, body);
+  return response.data;
```

Comment:
I think that if the POST returns a 400, `data` contains the errors. Should we maybe check the status code and raise an error in that case? :thinking:

Reply:
that already happens - the API client essentially throws for non-success status codes, see: https://github.com/open-formulieren/open-forms-sdk/blob/92fe7b7a840087d1b1aa20cc399c5229862efcf1/src/api.js#L126

in short, the call order: `post -> _unsafe -> apiCall -> throwForStatus`

Reply:
Ohh indeed!